packages/SwingSet/test/vat-admin/terminate/test-terminate.js

```diff
@@ -118,39 +157,55 @@ test.serial('terminate (dynamic, non-critical)', async t => {
     'v8',
     'kill',
     true,
-    'done exception kill (Error=false)',
+    [
+      'termP.catch Error: vatAdminService rejecting attempt to perform "terminateWithFailure"() on non-running vat "v8"',
```

Comment:
if we put these two long strings into short-named local variables, would `prettier` let us fit the `doTerminateNonCritical` call back onto a single line?

Reply:
Meh.  For now I'm happy just to tell `prettier` to shut up.  `prettier` is wrong so often I've become very blasé about that.